microsoft-365/compliance/office-365-service-encryption.md

```diff
@@ -20,9 +20,11 @@ description: "Summary: Understand data resiliency in Microsoft Office 365."
 
 In addition to using volume-level encryption, Exchange Online, Skype for Business, SharePoint Online, and OneDrive for Business also use Service Encryption to encrypt customer data. Service Encryption allows for two key management options:
 
-- Microsoft manages all cryptographic keys. (This option is currently available in SharePoint Online, OneDrive for Business, and Skype for Business.)
+## Microsoft managed keys: 
+Microsoft manages all cryptographic keys including the root keys for service encryption. This option is currently available in SharePoint Online and OneDrive for Business. This option is currently being rolled out for Exchange Online. This option provides default encryption using Microsoft mangaged keys unless the customer decides to onboard to using Customer Key. If Customer decides to leave Customer Key then the data will still stay encrypted using Microsoft managed keys. 
```

Comment:
Change "This option provides default encryption using Microsoft mangaged keys unless the customer decides to onboard to using Customer Key. If Customer decides to leave Customer Key then the data will still stay encrypted using Microsoft managed keys." to -
Microsoft managed keys provide default service encryption unless you decide to onboard using Customer Key. If, at a later date, you decide to stop using Customer Key without following the data purge path, then your data stays encrypted using the Microsoft managed keys. Your data is always encrypted at this default level at a minimum.